File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let maxLength = 0
    for (let i = 0; i < sentences.length; i++) {
        maxLength = Math.max(maxLength,sentences[i].split(" ").length)        
    }
    return maxLength
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
       int maxLength = 0;
    for (int i = 0; i < sentences.length; i++) {
        maxLength = Math.max(maxLength,sentences[i].split(" ").length);        
    }
    return maxLength;
        
    }
}